architecture/2024-08-controller-component-threat-model.md

```diff
@@ -0,0 +1,186 @@
+# Title
+
+- **Author**: ytimocin
+
+## Overview
+
+This document aims to provide a threat model for the Radius Controller component. In this threat model, you will find potential security threats to the Controller component, an important part of Radius, and possible mitigations to the risks they pose. The document includes an analysis of the system, its assets, security objectives, identified threats, and recommended security measures to protect the system.
```

Comment:
What's a `security objective`? The term is new to me.

Reply:
Removed this part.